observability/network_observability/network-observability-scheduling-resources.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="network-observability-scheduling-resources"]
+= Scheduling resources
+include::_attributes/common-attributes.adoc[]
+:context: network_observability_scheduling
+
+include::modules/network-observability-nodes-taints-tolerations.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations[Understanding taints and tolerations]
+* link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Kubernetes documentation - Assign Pods to Nodes]
```

Comment:
```suggestion
* link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Assign Pods to Nodes] (Kubernetes documentation)
```
New guidance for links: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences